bower.json

```diff
@@ -21,14 +21,14 @@
   ],
   "dependencies": {
     "polymer": "Polymer/polymer#0.8-preview",
-    "layout": "Polymer/layout",
-    "paper-styles": "PolymerLabs/paper-styles"
+    "paper-styles": "PolymerElements/paper-styles"
```

Comment:
Is it possible to depend on `polymerelements/paper-styles#^0.8.0`? This way we don't have to automatically update it again until `0.9.0` lands.